pkg/handlers/search/search_v2.go

```diff
@@ -60,15 +60,18 @@ func SearchDSL(c *gin.Context) {
 	}
 
 	if indexName != "" {
-		idx, _ := core.ZINC_INDEX_LIST.Get(indexName)
-		storageSize := idx.GetStats().StorageSize
-		eventData := make(map[string]interface{})
-		eventData["search_type"] = "query_dsl"
-		eventData["search_index_storage"] = idx.GetStorageType()
-		eventData["search_index_size_in_mb"] = storageSize / 1024 / 1024
-		eventData["time_taken_to_search_in_ms"] = resp.Took
-		eventData["aggregations_count"] = len(query.Aggregations)
-		core.Telemetry.Event("search", eventData)
+		// TODO: adapt this to allow strings.Split(indexName, ",") slice
+		idx, ok := core.ZINC_INDEX_LIST.Get(indexName)
```

Comment:
There is a TODO

Reply:
That should come in another PR, this one is large enough already.